packages/apps/reputation-oracle/server/src/modules/abuse/abuse.service.ts

```diff
@@ -207,7 +223,7 @@ export class AbuseService {
         abuseEntity.status = AbuseStatus.NOTIFIED;
         await this.abuseRepository.updateOne(abuseEntity);
       } catch (err) {
-        this.logger.error(`Error sending abuse: ${err.message}`);
+        this.logger.error(`Error sending abuse`, { err });
```

Comment:
```suggestion
        this.logger.error(`Error sending abuse`, {
          error: err,
          abuseId: abuseEntity.id
        });
```
This way you have all necessary info in log and don't have to put it to error